lib/documents_tagger.rb

```diff
@@ -32,10 +32,22 @@ def mapped_taxons(document_type)
     document_type.camelize.constantize.new.taxons
   end
 
+  def get_paged_editions_enum(params = {})
+    enum = Services.publishing_api.get_paged_editions(params)
+    Enumerator.new do |yielder|
+      enum.each do |response|
+        results = response.to_h.fetch('results', [])
+        results.each do |result|
+          yielder << result
+        end
```

Comment:
I'd perhaps consider getting rid of the `results` variable. Chaining the methods directly together may be easier to read, e.g.

```ruby
        response
          .to_h
          .fetch('results', [])
          .each { |result| yielder << result }
```